templates/create-daml-app/ui/src/config.ts

```diff
@@ -26,11 +26,3 @@ export const httpBaseUrl =
   deploymentMode === DeploymentMode.PROD_DABL
   ? `https://api.projectdabl.com/data/${ledgerId}/`
   : undefined;
-
-// Unfortunately, the development server of `create-react-app` does not proxy
-// websockets properly. Thus, we need to bypass it and talk to the JSON API
-// directly in development mode.
-export const wsBaseUrl =
```

Comment:
I don't think this is needed anymore as the DamlLedger context thing seems to default to where the page is running (normally `localhost::3000`) and the proxy can now be configured to work for websockets